docs/source/index.rst

```diff
@@ -523,8 +522,10 @@ Polling is not subject to handle state with HTTP status codes (e.g. 304)
                        ceph.conf sections (only global, mon, osd, rgw, mds
                        allowed) to keys and values. Anything defined in this
                        mapping will override existing settings.
-   :<json object devices: (required) A mapping of OSD device to Journal
-                          like device: {"device": "journal"}.
+   :<json object or array devices: (required) A mapping of OSD device to Journal
+                          like device: {"device": "journal"} (when the journal
+                          is separate from the OSD) or ["/dev/sdb", "/dev/sdc"]
+                          for collocated OSDs and Journals.
```

Comment:
Does this allow multiple uses of the same physical device? i.e. I have an NVMw device which can take up to twelve journals at /dev/sdc and I want to map /dev/sdd thru /dev/sdn to it? will it handle the detection of previously placed journals?